src/main/java/com/github/manolo8/darkbot/gui/MapDrawer.java

```diff
@@ -356,8 +356,11 @@ private void drawDynamicEntities(Graphics2D g2) {
         for (Npc npc : npcs) drawEntity(g2, npc.locationInfo.now, npc.npcInfo.kill);
 
         for (Ship ship : ships) {
+            Location loc = ship.locationInfo.now;
             g2.setColor(ship.playerInfo.isEnemy() ? ENEMIES : ALLIES);
             drawEntity(g2, ship.locationInfo.now, false);
+            if (config.MISCELLANEOUS.DISPLAY.SHOW_NAMES && !config.MISCELLANEOUS.DEV_STUFF)
```

Comment:
Why does dev stuff disable showing names? They should be kept separate settings IMO

Reply:
I thought dev stuf shows also ids of ships